api/db/migrations/20250808075247_add-created-at-and-updated-at-on-combined-course-participations.js

```diff
@@ -0,0 +1,19 @@
+const TABLE_NAME = 'combined_course_participations';
+const CREATED_AT_COLUMN = 'createdAt';
+const UPDATED_AT_COLUMN = 'updatedAt';
+
+const up = async function (knex) {
+  await knex.schema.table(TABLE_NAME, (table) => {
+    table.dateTime(CREATED_AT_COLUMN).notNullable().defaultTo(knex.fn.now());
```

Comment:
Pour info tu peux faire un truc du genre et ça te fait les deux colonnes bien nommées etc  

```suggestion
    table.timestamps(false, true, true);
```

Reply:
Trop bien je connaissais pas, je fais la modif ça me permettra de m'en rappeler pour les prochaines fois ! Merci 🫶